pkgs/run-sotest/src/run_sotest/poll_app.py

```diff
@@ -0,0 +1,33 @@
+#######################################
+#                                     #
+#  Copyright Cyberus Technology GmbH  #
+#           All rights reserved       #
+#                                     #
+#######################################
+
+import argparse
+
+from .poll_test_run import poll_test_run
+
+
+def parse_cmdline_args():
+    """Command line parsing"""
+    parser = argparse.ArgumentParser(description="Sotest Test Creator")
+    parser.add_argument("sotest_url", help="Base URL to SoTest's web UI")
```

Comment:
I was briefly confused because this is the only positional argument. Any reason this can't be `--sotest_url`?

Reply:
The idea is to keep this consistent with the `run_sotest` app that also has the URL (and the project config) as a positional argument. Would it be confusing if the usage of the two was different?

Reply:
Hm, I don't think it would make a big difference because not a lot of people actually use this tool standalone. I won't insist on a particular version, let's see what others think.